norent/schema.py

```diff
@@ -242,7 +242,23 @@ def perform_mutate(cls, form, info: ResolveInfo):
 @schema_registry.register_mutation
 class NorentFullName(NorentScaffoldingOrUserDataMutation):
     class Meta:
-        form_class = forms.FullName
+        form_class = forms.FullLegalName
+
+    deprecation_reason = "Use NorentFullLegalName instead."
+
+    @classmethod
+    def perform_mutate_for_authenticated_user(cls, form, info: ResolveInfo):
```

Comment:
I wasn't sure if the deprecated class still needed its full function definitions, but assumed it did and left it.